build.sbt

```diff
@@ -136,14 +136,17 @@ lazy val graphz = (project in file("graphz"))
   ).dependsOn(shared)
 
 lazy val casper = (project in file("casper"))
+  .configs(IntegrationTest extend Test)
   .settings(commonSettings: _*)
   .settings(rholangSettings: _*)
   .settings(
     name := "casper",
+    Defaults.itSettings,
     libraryDependencies ++= commonDependencies ++ protobufLibDependencies ++ Seq(
       catsCore,
       catsMtl,
-      monix
+      monix,
+      "org.scalacheck" %% "scalacheck" % "1.14.0" % "it"
```

Comment:
We already have scalacheck in `Dependencies.scala`. Let's not introduce a second one